### PR TITLE
Add RPMB simulation support on host side

### PR DESCRIPTION
**Pull request by James-Zzz**
rpmb_dev application listens to socket which connected to QEMU guest,
it handles data sent from QEMU guest, and operate RPMB_DATA on demand.
New virtio serial device would be exposed to QEMU guest, RPMB service
communicated with this virtio port for event handling.

Tracked-On: OAM-91232
Signed-off-by: Zhong,Fangjian <fangjian.zhong@intel.com>

### CODE REVIEW

File: groups/device-specific/caas/start_android_qcow2.sh [hunk 1]
```diff
@@ -404,23 +440,27 @@ function launch_hwrender(){
 
 	setup_usb_vfio_passthrough remove
 	cleanup_vsock_host_utilities
+	cleanup_rpmb_sock
```
Comment (yang8621) [inline groups/device-specific/caas/start_android_qcow2.sh:443]: if run kill -9 pidof qemu, will it be executed?

Comment (James-Zzz) [reply groups/device-specific/caas/start_android_qcow2.sh:443]: it should be executed if qemu is killed either by signaled with kill -9 or ctrl + c.
Need to be validated.

Comment (James-Zzz) [reply groups/device-specific/caas/start_android_qcow2.sh:443]: Update this conversation after validation, function cleanup_rpmb_sock would be executed on following 3 actions:
1. QEMU is killed by signaled with kill -9
2. QEMU process is killed by ctrl + c
3. QEMU instance is closed by clicking close button on QEMU GUI

**Comment by James-Zzz** [comment]
New PR created.

**Pull request closed without merging**